rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -1,7 +1,9 @@
+mod commit_switcher;
 mod utils;
```

Comment:
I think we generally put mods after uses.

Reply:
Done!